src/app/TestEventTriggerDelegate.h

```diff
@@ -0,0 +1,52 @@
+/*
+ *    Copyright (c) 2022 Project CHIP Authors
+ *
```

Comment:
supernit: standard license text has a ` *    All rights reserved.` right under copyright.

Mentioning it because I saw a recent PR adding it ...